discussion_agents/cog/functional/react.py

```diff
@@ -7,11 +7,14 @@
 from discussion_agents.cog.prompts.react import (
     REACT_INSTRUCTION,
     REACT_WEBTHINK_SIMPLE6_FEWSHOT_EXAMPLES,
+    REACT_WEBTHINK_SIMPLE3_FEVER_EXAMPLES
```

Comment:
if the variable to _build_agent_prompt is examples, I guess we won't need to import these examples, but they will be imported somewhere else when _build_agent_prompt is called